app/src/main/shared/components/Form.tsx

```diff
@@ -91,6 +92,12 @@ export class Form extends React.Component<FormProps, FormState> {
         });
     }
 
+    clearSuccess(){
+        this.setState({
+            hasSuccess: false
+        })
+    }
+
```

Comment:
unrelated to this ticket, but it's nice if the user can dismiss the success alert